testplan/web_ui/testing/src/Toolbar/Toolbar.js

```diff
@@ -503,14 +505,37 @@ const getToggledButtonStyle = (status) => {
  * Get the metadata from the report and render it as a table.
  */
 const getInfoTable = (report) => {
+  
+
   if (!report || !report.information) {
     return "No information to display.";
   }
   const infoList = report.information.map((item, i) => {
+    const linkifyIgnore = ['user', 'command_line_string',
+    'python_version', 'hostname', 'start', 'end'];
+
+
+    let columnData = !linkifyIgnore.includes(item[0]) ? < td className = {
```

Comment:
probably an exact match would be better instead of includes

Reply:
The ternary operator is hard to read here, and you repeating the same <td>.

a couple of options you can put the content goes into the <td> into a variable (here you can use the ternary operator )which is either the <div> or just the string content. Alternatively the working out of the linkify thing can be moved to it's own method together with the list of ignored fields, and just use that method in the template.